public/js/use.js

```diff
@@ -288,7 +288,7 @@ function setupUse(params) {
           var image = new Image();
           image.src = reader.result;
           image.onload = function () {
-            var resizedImage = (data.files[0]['size'] > 2000000) ? resize(image, 800) : this.src;
+            var resizedImage = resize(image, 800);
```

Comment:
Any image that is too small doesn't get resized.  The resize is done in the transform and the scale factor for small images is 1